app/schema_validation/definitions.py

```diff
@@ -48,10 +48,10 @@
 )
 
 https_url = {
```

Comment:
AC asked to reuse the `https_url` variable for the `callback_url` validation. This `https_url` variable to validate http urls is ONLY used for the callback url feature.